stix2/environment.py

```diff
@@ -152,3 +152,17 @@ def add(self, *args, **kwargs):
     def parse(self, *args, **kwargs):
         return _parse(*args, **kwargs)
     parse.__doc__ = _parse.__doc__
+
+    def created_by(self, obj):
+        """Retrieve the Identity refered to by the object's `created_by_ref`.
+
+        Args:
+            obj: The STIX object whose `created_by_ref` property will be looked
+                up.
+
+        Returns:
+            The STIX object's creator.
+        """
+
+        creator_id = obj.get('created_by_ref', '')
+        return self.get(creator_id)
```

Comment:
What should we do if the object has a `created_by_ref` but it can't be found via the configured DataSource? DataSource objects currently return None in this case, but I'm starting to wonder if that's the best approach; even if it is, I'm not sure if we want to make some distinction in this function.